templates/_macros/hero.jinja

```diff
@@ -2,14 +2,17 @@
 # title: Hero title text (required)
 # subtitle: Hero subtitle text
 # layout: layout of hero section. Options are '50/50', '50/50-full-width-image', '75/25', '25/75', 'fallback'
-# is_split_on_medium: whether the layout is split on tablet. Applies "-on-medium" to row class in 50/50 and 25/75 layouts. If false, the layout is stacked on tablet.
+# is_split_on_medium: whether the layout is split on tablet. Applies "-on-large" to row class in 50/50 and 25/75 layouts.
+#   If false, the layout is stacked on tablet.
+#   If true, the layout is split on tablet.
 # Slots
 # description: paragraph-style content below the title and subtitle
 # cta: call-to-action block below the description
 # image: slot for image content
+# signpost_image: slot for signpost (left column) image content in 25/75 layout. Required for 25/75 layout.
 
 {% macro hero(
-  title='',
+  title,
```

Comment:
FYI @bartaz I have removed the default value here, but still no errors are thrown if you don't pass in `title`. Seems required macro parameters aren't supported by Jinja currently. 

Nonetheless, I like not defining a default title for readability.